components/automate-chef-io/content/docs/infra-server.md

```diff
@@ -12,6 +12,12 @@ toc = true
     weight = 60
 +++
 
+{{% warning %}}
+
+We do not support using the Chef Automate 2 deployment service to deploy Chef Infra Server with Chef Manage.
```

Comment:
is there any way to get _Chef Manage_ closer to the beginning of the sentence? because up until that point it almost sounds like we are saying that we don't support deploying chef infra server with automate.

Reply:
Latest draft:
`Chef Manage cannot be used to deploy Chef Infra Server using Chef Automate`

Reply:
Or : `We do not support using Chef Manage with Chef Automate to deploy Chef Infra Server.`

Reply:
I would say something like "Chef Automate does not deploy Chef Manage alongside Chef Infra Server."